papis/commands/cache.py

```diff
@@ -1,5 +1,5 @@
 """
-This command allows the user to interact with the papis cache or papis database.
+This command allows the user to interact with the Papis cache or database.
```

Comment:
```suggestion
This command allows the user to interact with the Papis cache (database).
```
? I think most places use these as synonyms.

Reply:
Ok, that looks fine to me